packages/types/src/user.ts

```diff
@@ -6,4 +6,13 @@ export const UserSchema = type({
     userData: ScheduleSaveStateSchema,
 });
 
+export const AuthUserSchema = type({
+    id: 'string',
+    'userData?': ScheduleSaveStateSchema,
+    name: 'string',
+    email: 'string',
+    picture: 'string',
```

Comment:
Thoughts on calling this `pictureURL` instead of just `picture`?

Reply:
What about `image`? Maybe same problem, but I generally like simple names even if they're a bit ambiguous, just because there's not many different things it could be IMO.

Reply:
I guess simple could be fine here cause it's already defined as a string and normally you never see raw image data being passed around in web dev.